bundle/language_en.properties

```diff
@@ -1517,6 +1520,16 @@ angal.pricesothers.othm
 angal.pricesothers.pleaseselectanitemtodelete                                                          = Please select an item to delete
 angal.pricesothers.pleaseselectanitemtoedit                                                            = Please select an item to edit
 angal.pricesothers.thedatacouldnotbedeleted                                                            = The data could not be deleted
+angal.reduction.exam                                                                                   = Exam
+angal.reduction.examrate.col                                                                           = Exam rate
+angal.reduction.medical                                                                                = Medical
+angal.reduction.medicalrate.col                                                                        = Medical rate
+angal.reduction.operate.col                                                                            = Ope. rate
```

Comment:
Why did you write `Ope. rate` like this?

Reply:
it's how it is writhing in oh_dschang. will i change it???